db/migrate/20230601085414_create_result_links.rb

```diff
@@ -0,0 +1,16 @@
+class CreateResultLinks < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension 'citext' unless extension_enabled?('citext')
+
+    create_table :result_links do |t|
+      t.references :search_stat, null: false, foreign_key: true
+      t.integer :link_type, null: false
+      t.citext :url, null: false
+
+      t.timestamps default: -> { 'CURRENT_TIMESTAMP' }
```

Comment:
I think it's enough, did you face any problems with it? 🤔 
```suggestion
      t.timestamps
```

Reply:
Umm, no. Took idea from another codebase. Thought that's how timestamps should be initialized in rails.